batch-import/register_batch_source.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
+
+# Use this script to register a batch-input source with your Camio account. 
+# You need to know the following information to use this script
+# 1 - Device ID of the Camio-Box you plan on using this batch-input source with
+# 2 - 
+
+function register_batch_camera {
+    # args
+    # 1 - testing/production
+    # 2 - device-id
+    # 3 - local_camera_id
+    # 4 - cameraname
+
+    if [ $1 == "test" ]; then
+        : "${CAMIOTOKENTEST? Need to set CAMIOTOKENTEST environment variable to use this script in 'test' mode}"
+        : "${CAMIOIDTEST? Need to set CAMIOIDTEST environment variable to use this script in 'test' mode}"
+        url="https://test.camio.com/api/cameras/discovered"
+        token=$CAMIOTOKENTEST
+        userid=$CAMIOIDTEST
+    else
+        : "${CAMIOTOKENPROD? Need to set CAMIOTOKENPROD environment variable to use this script in 'prod' mode}"
+        : "${CAMIOIDPROD? Need to set CAMIOIDPROD environment variable to use this script in 'prod' mode}"
+        url="https://www.camio.com/api/cameras/discovered"
+        token=$CAMIOTOKENPROD
+        userid=$CAMIOIDPROD
+    fi
+    deviceid=$2
+    localcameraid=$3
+    cameraname=$4
+
+    cat <<EOF > /tmp/tmp_batch_import.json
+{
+    "$localcameraid": {
+    "device_id_discovering": "$deviceid",
+    "acquisition_method": "batch",
+    "discovery_history": {},
+    "device_user_agent": "CamioBox (Linux; virtualbox) Firmware:box-virtualbox-cam.201704120128-523def1681975a7edb19a082e82fa53e06
+    "user_id": "$userid",
+    "local_camera_id": "$localcameraid",
+    "name": "$cameraname",
+    "mac_address": "$localcameraid",
+    "maker": "",
+    "is_authenticated": false,
+    "rtsp_path": "",
+    "rtsp_server": "",
+    "should_config": false,
+    "default_values": {
+      "plan": {
+        "is_multiselect": false,
+        "options": [
+          {
+            "name": "Basic",
+            "value": "BASIC"
+          }
+        ]
+      }
+    },
+    "actual_values": {
+      "plan": {
+        "is_multiselect": false,
+        "options": [
+          {
+                        {
+            "name": "Baisc",
+            "value": "BASIC"
+          }
+        ]
+      },
+      "username": {
+        "options": [
+          {
+            "value": "admin"
+          }
+        ]
+      },
+      "password": {
+        "options": [
+          {
+            "value": "admin"
+          }
+        ]
+      }
+    },
+    "rtsp_url_pattern_python": ""
```

Comment:
I only have it here because (https://github.com/CamioCam/Camiolog-Web/issues/4637) is still unanswered. I need to know what fields are required to register a camera.